test/e2e-tests.sh

```diff
@@ -31,6 +31,10 @@ failed=0
 
 # Run upstream knative serving & eventing tests
 (( !failed )) && ensure_serverless_installed || failed=15
+
+# Run knative serving additional e2e tests
+(( !failed )) && run_serving_e2e_tests || failed=5
```

Comment:
What about changing the name of this function to `run_downstream_serving_e2e_tests` and the function two lines below to `run_upstream_knative_serving_e2e_and_conformance_tests` ? I'd like to differentiate them because right now the difference is not clear.
Maybe I'd even remove the `run_` prefix from all the functions that are called from this file. They seem redundant because functions such as `upstream_knative_serving_e2e_and_conformance_tests` or `knative_eventing_operator_tests` is understandable.

Reply:
Sounds very reasonable since I had trouble differentiating between `run_e2e_tests` and `run_knative_serving_e2e_and_conformance_tests' too when I first looked at it. Adding `upstream` and `downstream` will clear any confusion.